pkg/operator/vttablet/pod.go

```diff
@@ -87,6 +87,11 @@ func UpdatePod(obj *corev1.Pod, spec *Spec) {
 		key = strings.TrimLeft(key, "-")
 		vttabletAllFlags[key] = value
 	}
+	// Ensure that binary logs are restored to/from a location that all containers
+	// in the pod can access if no location was explicitly provided.
+	if _, ok := vttabletAllFlags["builtinbackup-incremental-restore-path"]; !ok {
+		vttabletAllFlags["builtinbackup-incremental-restore-path"] = vtDataRootPath
+	}
```

Comment:
What would happen if the path specified in `--builtinbackup-incremental-restore-path` is not accessible to all containers in the pod?

Reply:
I am guessing it is up to the user to set the same value on all components too? `mysqlctl`, `vttablet` and `vtbackup`

Reply:
The same thing that happens now to every user. It doesn't work. PITR does not generally work in the operator.